content/docs/zero/api-guide.md

```diff
@@ -0,0 +1,250 @@
+description: This page shows you how to create a route and policy with the Pomerium Zero API.
+---
+
+# Get Started With the Pomerium Zero API
+
+This guide walks you through creating your first route and policy with the Pomerium Zero API.
+
+:::tip
+
+Throughout this guide, you can refer to either the [**Zero API reference**](/docs/api) or the Zero API playground at `https://console.pomerium.app/openapi`.
+
+:::
+
+## 1. Authenticate
+
+### Get API User Token
+
+The Zero API requires authenticated access for both personal accounts and organizations. To send an authenticated request, first create an **API User** in the Zero Console:
+
+1. In the **Account** menu, select **API Tokens**
+1. Select **Add API User**
+1. Name the API user and select **Create**
+1. Copy the **API User Token** (keep it somewhere safe; you can't view it again)
+
+### Get ID token
+
+Now, exchange the API User Token for an ID token by sending a `POST` request to the `/token` endpoint. Make sure you replace `<API-USER-TOKEN>` with the one generated for you in the Zero Console:
+
+```curl
+curl --location 'https://console.pomerium.app/api/v0/token' \
+--header 'Content-Type: application/json' \
+--data '{
+  "refreshToken": "<API-USER-TOKEN>"
+}'
+```
+
+If your request was successful, you'll get a response with a temporary ID token:
+
+```json
+{
+  "expiresInSeconds": "3600",
+  "idToken": "<ID-TOKEN>"
+}
+```
+
+Copy the ID token and keep it somewhere safe. You won't be able to see it again, and you need it to make authenticated requests.
+
+## 2. Create a policy
+
+To create a policy, you need your organization ID and your namespace ID.
+
+### Get organization ID
+
+There are two types of organizations: **personal** and **professional**. When you create an account, you get a personal organization by default.
+
+To get your personal organization ID, send a `GET` request to the `/organizations` endpoint:
+
+```curl
+curl --location 'https://console.pomerium.app/api/v0/organizations' \
+--header 'Authorization: Bearer <ID-TOKEN>' \
+--data ''
+```
+
+If your request was successful, you'll get a response with your organization ID and other related metadata:
+
+```json
+[
+  {
+    "createdAt": "2024-01-17T20:07:47.794672Z",
+    "id": "bwjkRZwxbNXBQHHcJHphGSNBbxt",
+    "joinedAt": "2024-05-29T20:27:27.336939Z",
+    "name": "personal",
+    "organizationType": "personal",
+    "ownerUserId": "bMWxgvnRQfcRHjQvfMehbJhYFzB",
+    "role": "admin",
+    "updatedAt": "2024-01-17T20:07:47.794672Z"
+  }
+]
+```
+
+Copy your organization ID.
+
+### Get namespace ID
+
+In Pomerium Zero, each cluster is assigned its own namespace. To make changes to your cluster's configuration, you need your cluster's namespace ID.
+
+To get your cluster's namespace ID, send a `GET` request to the `/namespaces` endpoint, replacing `{organizationId}` with your own:
+
+```curl
+curl --location 'https://console.pomerium.app/api/v0/organizations/{organizationId}/namespaces' \
```

Comment:
```suggestion
curl --location 'https://console.pomerium.app/api/v0/organizations/{organizationId}/clusters' \
```

Maybe we should list clusters as this is the concept available in the API, while namespace is currently not part of the UI concepts; the cluster entity would have `namespaceId` that user could use in the subsequent calls to create routes and policies

Reply:
I'm incorporating this feedback and removing the previous payload example. I'll also remove the parent/child cluster relationship explanation.